### PR TITLE
changes

### PR DESCRIPTION
**Pull request by sundaramoorthy175**

### CODE REVIEW

File: ajaxthreepopup.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+			console.log(data);
+			showData.empty();
+			if (data.items.length){
+				$.fancybox.open(data.items);
+
+			}
+			});
+			showData.text('Loading the JSON file.');
+		});
+        $('#demotwo').click(function (){
```
Comment (arvindwill) [inline ajaxthreepopup.html:22]: need to avoid this block

File: ajaxthreepopup.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+			console.log(data);
+			showData.empty();
+			if (data.items.length){
+				$.fancybox.open(data.items);
+
+			}
+			});
+			showData.text('Loading the JSON file.');
+		});
+        $('#demothree').click(function (){
```
Comment (arvindwill) [inline ajaxthreepopup.html:34]: need to avoid this block

Comment (sundaramoorthy175) [reply ajaxthreepopup.html]: completed sir

File: ajaxthreepopup.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/fancyapps/fancybox@3.5.7/dist/jquery.fancybox.min.css" />
+	<script src="https://cdn.jsdelivr.net/gh/fancyapps/fancybox@3.5.7/dist/jquery.fancybox.min.js"></script>
+	
+	<script>
+	$(document).ready(function () {
+		$('.parent').click(function (){
+			var showData = $('#show-data');
+			var theID = $(this).attr('id');
+			if( theID == 'demo'){
+				var jsonFile = 'https://weblab.stutzen.xyz/json/jsonsample.json';
```
Comment (arvindwill) [inline ajaxthreepopup.html:14]: this path must be embedded in the HTML itself. Can use **data-** attribute.

Comment (sundaramoorthy175) [reply ajaxthreepopup.html]: completed.

**Pull request merged**